india_compliance/gst_india/doctype/purchase_reconciliation_tool/exporter.py

```diff
@@ -0,0 +1,216 @@
+import openpyxl
+
+# from openpyxl.formatting.rule import Rule
+# from openpyxl.styles import Alignment, Font, PatternFill
+# from openpyxl.styles.differential import DifferentialStyle
+# from openpyxl.utils import get_column_letter
+
+
+class ExcelExporter:
+    def __init__(self):
+        pass
+
+    def get_workbook(self, workbook=None):
+        if not workbook:
+            workbook = openpyxl.Workbook()
+
+        return workbook
+
+    def create_worksheet(self, *args, **kwargs):
+        """
+        Make xlsx file
+        :param workbook - Object of excel file/ workbook
+        :param sheet_name - name for the worksheet
+        :param filters - A data dictionary to added in sheet
+        :param merged_headers - A dict of List
+            @example: {
+                'label': [column1, colum2]
+            }
+        :param headers: A List of dictionary (cell properties will be optional)
+        :param data: A list of dictionary to append data to sheet
+        :param file_name: Name of excel file/workbook
+        """
+        # Create workbook if not passed or return existing workbook
+        wb = self.get_workbook(kwargs.get("workbook"))
```

Comment:
A separate function is better. Save it as self.wb so you do not have to pass this again.